presto-hive/src/main/java/com/facebook/presto/hive/PrestoS3FileSystem.java

```diff
@@ -141,6 +141,8 @@ public static PrestoS3FileSystemStats getFileSystemStats()
 
     private static final DataSize BLOCK_SIZE = new DataSize(32, MEGABYTE);
     private static final DataSize MAX_SKIP_SIZE = new DataSize(1, MEGABYTE);
+    private static final String PATH_SEPARATOR = "/";
```

Comment:
I personally don't think this is necessary, but you feel it improves readability, we can add a constant